### DISCUSSION TITLE
Milestone not showing closes Issues

### DISCUSSION THREAD

**Discussion by lvnkmn** [community/community#22947] [New to GitHub]
<p>My milestones stopped showing the closed issues. It does show how many closed issues it should contain. But it doesn’t actually show them.</p>
<p>I’ve tried this in both Safari and Chrome, doesn’t work in either of them. </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/a31c86e3789f69b315354eb9d03646b4dfd83b36.png" data-download-href="/uploads/short-url/ngWWFApMwFzCB0K5SxRZCxWv46G.png?dl=1" title="Screen Shot 2018-05-14 at 14.04.13.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/2656075/181073787-140198cf-0086-4ac9-9ef7-c22f4160a9c5.png" alt="Screen Shot 2018-05-14 at 14.04.13.png" data-base62-sha1="ngWWFApMwFzCB0K5SxRZCxWv46G" width="690" height="218" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/a31c86e3789f69b315354eb9d03646b4dfd83b36_2_690x218.png, https://aws1.discourse-cdn.com/github/optimized/1X/a31c86e3789f69b315354eb9d03646b4dfd83b36_2_1035x327.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/1X/a31c86e3789f69b315354eb9d03646b4dfd83b36_2_1380x436.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/a31c86e3789f69b315354eb9d03646b4dfd83b36_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2018-05-14 at 14.04.13.png</span><span class="informations">2124×672 42.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by gregvanl** [comment]
<p>Seeing the same issue on Windows 10 with Edge and Chrome. Seems like a recent regression.</p>

**Comment by ThirdEyeRose** [comment, marked as answer]
<p>Hi <a class="mention" href="/mennolovink">@mennolovink</a>,</p>
<p>This post was moved to a different board that fits your topic of discussion a bit better. This means you’ll get better engagement on your post, and it keeps our Community organized so users can more easily find information.</p>
<p>As you’ll notice, your Topic is now in the How to use Git and GitHub board. No action is needed on your part; you can continue the conversation as normal here.</p>
<p>Additionally, it sounds like our Engineering team has recently deployed a fix to this particular issue. Are you still seeing the problem?</p>
<p>Let me know if you have any other questions or if I can help with anything else.</p>
<p>Cheers!</p>

**Comment by lvnkmn** [comment]
<p>Nope, today I didn’t see this anymore.</p>
